metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_schema.py

```diff
@@ -3,6 +3,7 @@
 from datetime import datetime
 from typing import Dict, List, Optional
 
+import pandas as pd
```

Comment:
nit: this should be a typing-only dep - see mypy's docs on `TYPE_CHECKING`

Reply:
also I believe this PR https://github.com/acryldata/datahub-classify/pull/6 is going to remove the pandas dep from datahub-classify

Reply:
We are using pandas to read from snowflake cursor and translate to pandas dataframe in snowflake- https://docs.snowflake.com/en/user-guide/python-connector-pandas.html#migrating-to-pandas-dataframes. so this is required dependency for snowflake connector. I am not sure how converting this to typing-only dep will be useful.

Reply:
got it - in this case it's fine then, but I don't want to have a hard dep on pandas when we add classification to other sources

Reply:
Makes sense. I believe that would need refractor in [classification_mixin](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/glossary/classification_mixin.py#L105) as well. Maybe, we can replace `sample_data` input param with a callable function that takes column as input and returns list-like values structure as output ?

Reply:
Sure, although I suspect a dict {column name -> list[sample values]} is good enough for this

Reply:
seems fair.